.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+  - "node"
+
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.2.1
```

Comment:
https://docs.travis-ci.com/user/caching/#yarn-cache

looks like you just want `cache: yarn`